yarn.lock

```diff
@@ -3534,6 +3534,51 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@sinonjs/commons@npm:^2.0.0":
```

Comment:
Question: can you help me understand why we had to add sinonjs?

Reply:
Sinon is needed to mock/stub the `window.navigator.writeText` functionality, which is not available in the context of Ember testing (the "click" is synthetic, for this reason the browser throws an error when the clipboard is invoked).

See code in this set of tests: https://github.com/hashicorp/design-system/pull/1626/files#diff-8e3f45af3dda12cba8ca2e98080fe6fddf4023f8df76520a64d3c8da68bbde21